hip-0000-exchange-rate-system-contract.md

```diff
@@ -0,0 +1,128 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Exchange rate system contract
+author: Michael Tinker (@tinker-michaelj)
+working-group: Danno Ferrin (@shemnon)
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 05-12-2022
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/474
+---
+
+## Abstract
+
+We propose a new precompiled contract (or _system contract_) to convert between tinybars and tinycents at the network's 
+active exchange rate in system file `0.0.112`; that is, at the exchange rate the network is using to calculate fees. 
+
+This precompile would **NOT** be a live price oracle appropriate to use in, for example, a DEX client contract.
```

Comment:
I might include some verbiage about how this exchange rate is used to convert from the fee schedule to fees in tinybars.